src/psyclone/tests/psyGen_test.py

```diff
@@ -2512,7 +2512,7 @@ def test_modified_kern_line_length(kernel_outputdir, monkeypatch):
     assert os.path.isfile(filepath)
     # Check that the argument list is line wrapped as it is longer
     # than 132 characters.
-    assert "undf_w3,&\n&map_w3)\n" in open(filepath).read()
+    assert "map_w2, &\n&ndf_w3" in open(filepath).read()
```

Comment:
Line broken in different location due to additional spaces?

Reply:
Yes. It took me a while to figure out why the test wasn't passing :)